deploy/kubernetes/resources/gameon-ingress.json

```diff
@@ -0,0 +1,30 @@
+{
+  "apiVersion": "extensions/v1beta1",
+  "kind": "Ingress",
```

Comment:
@mckeeh3 isn't the plan to use a shared ingress?

Reply:
yes, the plan is to use a shared ingress. Should this file be removed and we add the ingress paths to the ingress file in the chirper project?

Reply:
We could either modify this one or put the ingress config into a new project.